source/manage/server-admin-gui.rst

```diff
@@ -24,19 +24,28 @@ Press *Apply changes* to store any changes.
 Metrics resolution
 ******************
 
-PMM collects metrics at three different time intervals.
-
+Metrics are collected at three intervals representing low, medium and high resolutions.
 Short time intervals are regarded as high resolution metrics, while those at longer time intervals are low resolution.
 
-The :guilabel:`Metrics Resolution` slider lets you choose from three preset combinations of intervals corresponding to high, medium, and low resolution (short, medium, and long collection periods).
+The default values are:
+
+- Low: 60 seconds
+- Medium: 10 seconds
+- High: 5 seconds
+
+The *Metrics Resolution* slider lets you choose from three preset combinations of intervals corresponding to high, medium, and low resolution (short, medium, and long collection periods).
+
+The slider tool-tip shows the collection time corresponding to each resolution setting.
+
+- Setting the slider to *Low* increases the time between collection, resulting in low-resolution metrics.
```

Comment:
"resulting in data being collected less frequently, thus reducing the storage cost but at a tradeoff of resolution" or something like thist

Reply:
Can I instead add to the note under this section a mention of the tradeoff: hi-res = more resources but more accuracy, low-res = less resources, less accuracy.